proposals/0000-language-editions.rst

```diff
@@ -0,0 +1,1144 @@
+|``EmptyDataDeriving``                          |   X   |       |  Y   |  Y  |  Y   |          |        |       |        |        |     |   |  |      |        |        |  |  |  |                                  |
++-----------------------------------------------+-------+-------+------+-----+------+----------+--------+-------+--------+--------+-----+---+--+------+--------+--------+--+--+--+----------------------------------+
+|``ExistentialQuantification``                  |   X   |       |      |     |  Y   |   X      |        |       |        |        |     |   |  |      |        |        |  |  |  | [#]_                             |
++-----------------------------------------------+-------+-------+------+-----+------+----------+--------+-------+--------+--------+-----+---+--+------+--------+--------+--+--+--+----------------------------------+
+|``ExplicitForAll``                             |   X   |   Y   |  Y   |  Y  |  Y   |          |        |       |        |        |     |   |  |      |        |        |  |  |  |                                  |
++-----------------------------------------------+-------+-------+------+-----+------+----------+--------+-------+--------+--------+-----+---+--+------+--------+--------+--+--+--+----------------------------------+
+|``ExplicitNamespaces``                         |       |   Y   |  Y   |  Y  |  Y   |          |        |       |        |        |     |   |  |      |        |        |  |  |  |                                  |
++-----------------------------------------------+-------+-------+------+-----+------+----------+--------+-------+--------+--------+-----+---+--+------+--------+--------+--+--+--+----------------------------------+
+|``ExtendedDefaultRules``                       |       |   Y   |      |     |      |          |        |       |        |        |     |   |  |      |        |        |  |  |  |*flag*                            |
++-----------------------------------------------+-------+-------+------+-----+------+----------+--------+-------+--------+--------+-----+---+--+------+--------+--------+--+--+--+----------------------------------+
+|``ExtendedLiterals``                           |       |       |      |     |  Y   |          |        |       |   X    |        |     |   |  |      |        |        |  |  |  |                                  |
++-----------------------------------------------+-------+-------+------+-----+------+----------+--------+-------+--------+--------+-----+---+--+------+--------+--------+--+--+--+----------------------------------+
+|``FieldSelectors``                             |   X   |       |      |     |      |          |        |   X   |        |        |     |   |  |      |        |        |  |  |  |*flag*                            |
```

Comment:
Yes, that's what I mean. No field selectors by default, but you can have them in `Classic` mode or by request. (Actually, it would be very cool if we could have `deriving selectors` or something.) Yes, it's a bold move, but I think the right one.